src/main/java/org/folio/service/inventory/InventoryManager.java

```diff
@@ -355,8 +356,17 @@ private static void handleHoldingsError(String holdingId, Throwable throwable) {
   }
 
   public Future<List<JsonObject>> getHoldingsByIds(List<String> holdingIds, RequestContext requestContext) {
+    return getHoldingsByIds(holdingIds, requestContext, this::fetchHoldingsByHoldingIds);
+  }
+
+  public Future<List<JsonObject>> getHoldingsByIdsWithoutVerification(List<String> holdingIds, RequestContext requestContext) {
+    return getHoldingsByIds(holdingIds, requestContext, this::fetchHoldingsByHoldingIdsWithoutVerification);
+  }
+
+  public Future<List<JsonObject>> getHoldingsByIds(List<String> holdingIds, RequestContext requestContext,
+                                                   org.folio.service.orders.utils.HelperUtils.BiFunctionReturningFuture<List<String>, RequestContext, List<JsonObject>> biFunction) {
```

Comment:
can we move to import section, it is being difficult to read

Reply:
unfortunatelly no, because we alredy refer to HelperUtils from another package